stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/experimental/channel/logic/ChannelLogic.kt

```diff
@@ -274,6 +277,54 @@ internal class ChannelLogic(
         .maxOrNull()
         ?: NEVER.time
 
+    /**
+     * Returns instance of [WatchChannelRequest] to obtain older messages of a channel.
+     *
+     * @param limit Message limit in this request.
+     * @param baseMessageId Message id of the last available message. Request will fetch messages older than this.
+     */
+    internal fun olderWatchChannelRequest(limit: Int, baseMessageId: String?): WatchChannelRequest =
+        watchChannelRequest(Pagination.LESS_THAN, limit, baseMessageId)
+
+    /**
+     * Returns instance of [WatchChannelRequest] to obtain newer messages of a channel.
+     *
+     * @param limit Message limit in this request.
+     * @param baseMessageId Message id of the last available message. Request will fetch messages newer than this.
+     */
+    internal fun newerWatchChannelRequest(limit: Int, baseMessageId: String?): WatchChannelRequest =
+        watchChannelRequest(Pagination.GREATER_THAN, limit, baseMessageId)
+
+    /**
+     * Creates instance of [WatchChannelRequest] according [Pagination].
```

Comment:
Perhaps "Creates instance of [WatchChannelRequest] according **to** [Pagination]." would be more accurate?